contracts/transfer/src/wasm/internal.rs

```diff
@@ -72,11 +72,14 @@ impl TransferContract {
         address: &ContractId,
         pk: &PublicKey,
     ) -> Result<Message, Error> {
-        self.message_mapping
+        let mut mm = self.message_mapping.inner_mut()?;
+
+        let x = mm
```

Comment:
Also, when ready to merge, return `mm.get_mut...` directly instead of using intermediate `x`